### PR TITLE
Add signature verification to / endpoint

### PR DESCRIPTION
**Pull request by benvinegar**
Implementing this check to verify that webhook POST requests from Sentry are genuine: https://docs.sentry.io/workflow/integrations/integration-platform/webhooks/#verifying-the-signature

Note: I haven't actually verified this verification works. @LearningNerd since you have an end-to-end testing environment working, can you try this out?

### CODE REVIEW

**Comment by LearningNerd** [comment]
Not working for me; all requests get marked unverified. From ngrok at least, the headers were all lowercase -- but after changing that, still no luck.

All the request headers coming in for the issue-created webhook look like this:
```
host: 'b2e24914.ngrok.io',
  'sentry-trace': 'blabla',
  'accept-encoding': 'gzip, deflate',
  accept: '*/*',
  'user-agent': 'sentry/10.0.0.dev0 (https://sentry.io)',
  'request-id': 'blablalba',
  'sentry-hook-signature': 'blablablabla',
  'content-type': 'application/json',
  'sentry-hook-resource': 'issue',
  'sentry-hook-timestamp': '1571188825',
  'content-length': '912',
  'x-forwarded-for': 'some.ip.address.here' }
```
Tried this out but no luck:
https://stackoverflow.com/questions/48843112/how-to-compare-hmac-sha256-signatures-from-request-body-in-nodejs

**Comment by benvinegar** [comment]
Okay, I debugged this quickly – the fundamentals were correct, but Express downcases the header name.

**Comment by benvinegar** [comment]
Success! (The issue below was assigned via the app, after verifying the signature.)

image

Aside: we should edit the sample code on verifying signatures to include this Node example. cc @scefali @MeredithAnya

File: verify.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ function verifySignature(request) {
   const hmac = crypto.createHmac("sha256", process.env.SENTRY_API_SECRET);
   hmac.update(JSON.stringify(request.body), 'utf8');
   const digest = hmac.digest('hex');
-  return hmac.digest('hex') === request.headers["sentry-hook-signature"];
+  return digest === request.headers["sentry-hook-signature"];
```
Comment (LearningNerd) [inline verify.js]: 🤦‍♀ derp. totally missed that too.

**Pull request merged**